src/pybind/mgr/dashboard/frontend/src/app/shared/models/cd-table-fetch-data-context.ts

```diff
@@ -23,15 +23,22 @@ export class CdTableFetchDataContext {
   }
 
   toParams(): HttpParams {
+    if (this.pageInfo.offset == NaN) {
```

Comment:
I would prefer [isNan](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN#using_isnan)
```suggestion
    if (Number.isNaN(this.pageInfo.offset)) {
```

Reply:
done!